### DISCUSSION TITLE
Marker interaction in stereo mode

### DISCUSSION THREAD

**Discussion by lukeIam** [mistic100/Photo-Sphere-Viewer#1119] [General]
Hi 👋,

nice library!
One question: does somebody knows a way how to let a user interact with markers in stereo mode?

I thought about adding a dot in the center of the users view + when the view does not move show a loading circle. After some time of non-movement the click event of the marker "near" of the "cursor" dot is invoked.
But I was unable to implement this in a stereo view.

Any thoughts?

**Comment by jywarren** [comment]
I like this idea a lot. I am interested in whether `enterMarker` and `leaveMarker` are triggered when the user turns directly towards those coordinates? https://photo-sphere-viewer.js.org/plugins/markers.html#events

If so, I might try something similar like, cueing an audio track to play. I imagine one could measure how long they have been hovering, and trigger something then as well. It's like the Photosphere making UI on Android, where you have to find the dots and hover over them for a second or two each to capture the full photosphere.

**Comment by mistic100** [comment]
https://github.com/mistic100/Photo-Sphere-Viewer/issues/1307
